### PR TITLE
Yogstation Drinks Expansion 1: More teas

### PR DESCRIPTION
**Pull request by AlvCyktor**
As the title said, this PR will expand on tea related items for bartending.

# Document the changes in your pull request

Adds more tea, tea related accessories + a tea machine.

# Changelog

:cl:  
rscadd: Tea
wip: This is work in progress  
/:cl:

### CODE REVIEW

**Comment by Anvilman6** [comment]
👀

**Comment by Marmio64** [comment]
just in time for British language

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 1]
```diff
@@ -370,6 +370,112 @@
...
+	name = "Rooibos Tea"
+	description = "A reddish-brown tea with needles submerged at the bottom."
+	color = "bc6f63" // rgb: 188, 111, 99
+	nutriment_factor = 0
+	taste_description = "slightly sweet and nutty"
+	glass_icon_state = "teaglass"
+	glass_name = "glass of rooibos tea"
+	glass_desc = "You can see small pins and needles swirling at the bottom."
+
+/datum/reagent/consumable/tea/red/on_mob_life(mob/living/carbon/M)
```
Comment (SomeguyManperson) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: if these are all gonna have the same effects as the original tea reagent you don't need to add them, they'll be inherited.
tldr is if /datum/reagent/consumable/tea has a variable or proc anything /datum/reagent/consumable/tea/whatever will default to using that variable or proc's value unless specifically changed

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 2]
```diff
@@ -370,6 +370,70 @@
...
+/datum/reagent/consumable/tea/red
+	name = "Rooibos Tea"
+	description = "A reddish-brown tea with needles submerged at the bottom."
+	color = "bc6f63" // rgb: 188, 111, 99
+	taste_description = "slightly sweet and nutty"
+	glass_icon_state = "teaglass"
+	glass_name = "glass of rooibos tea"
+	glass_desc = "You can see small pins and needles swirling at the bottom."
+
+/datum/reagent/consumable/tea/red/on_mob_life(mob/living/carbon/M)
```
Comment (SomeguyManperson) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]:
```suggestion
```
tldr do this for the lot of them

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 3]
```diff
@@ -370,6 +370,70 @@
...
+	.=..()
```
Comment (SomeguyManperson) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]:
```suggestion
```

**Comment by AlvCyktor** [comment]
I don't have time for this. Will close now
